reclass/datatypes/parameters.py

```diff
@@ -134,19 +153,42 @@ def _merge_dict(self, cur, new, path):
             ret.update(new)
             return ret
 
+        ovrprfx = Parameters.DICT_KEY_OVERRIDE_PREFIX
+
         for key, newvalue in new.iteritems():
-            ret[key] = self._merge_recurse(ret.get(key), newvalue,
-                                           path.new_subpath(key))
+            if key.startswith(ovrprfx) and not initmerge:
+                ret[key.lstrip(ovrprfx)] = newvalue
+            else:
```

Comment:
I agree with this in principle but in practice it's a mite hairier... in reality processed parameter keys never actually reach the user because merge is called outside init in core. I've tested this by creating a node with no classes and "tilde-params". Furthermore if we strip them in the __init__ merge they do not exist as a signalling character during the "proper" merge, in which case it merges as normal (contrary to expectations).

I'm not averse to reformulating this conditional but I'd maybe like to see something that you think would work. I futzed with this a lot and there's some delicate interplay going between Parameters.__init__ and Core._nodeinfo